test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ServiceBusEndToEndTests.cs

```diff
@@ -125,9 +130,17 @@ private void ServiceBusEndToEndInternal()
                 TypeLocator = new FakeTypeLocator(this.GetType())
             };
 
-            _namespaceManager = NamespaceManager.CreateFromConnectionString(config.ServiceBusConnectionString);
+            ServiceBusConfiguration serviceBusConfig = new ServiceBusConfiguration();
+            config.UseServiceBus(serviceBusConfig);
```

Comment:
This line shows the new extensibility model in action. The user calls extension methods for the extensions they want to use.